projects/plugins/crm/includes/ZeroBSCRM.MailTracking.php

```diff
@@ -455,7 +455,7 @@ function zeroBSCRM_get_email_status($ID){
 	$sql = $wpdb->prepare("SELECT zbsmail_active FROM " . $ZBSCRM_t['system_mail_templates'] . " WHERE zbsmail_id = %d", $ID);
 	$r = $wpdb->get_results($sql);
 
-	if ( isset( $r[0] ) && $r[0]->zbsmail_active === 1 ) {
+	if ( isset( $r[0] ) && $r[0]->zbsmail_active == 1 ) { // phpcs:ignore
```

Comment:
```suggestion
	if ( isset( $r[0] ) && $r[0]->zbsmail_active == 1 ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual -- strict comparison breaks functionality.
```
I'd add the name of the sniff so that it doesn't ignore everything else on the line, and in cases like this explain briefly why we're ignoring it.

Reply:
Cool! I copied from someone the lazy way to satisfy the linter.